### ISSUE TITLE
Combo-box : Disable combo box having color contrast issue 

### ISSUE THREAD

**Issue by MdAmeerMind** [microsoft/fluentui#13224]
### Describe the issue:
Fast pass fail for disable combo-box

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/mdameer/pen/ExVOGpz?editors=0100

#### Actual behavior:
Chrome Fast pass fail for disable combo-box
<img width="956" alt="Combobox-Disable" src="https://user-images.githubusercontent.com/48435553/82294164-544d8a00-99cb-11ea-999a-fb04b7723559.PNG">

#### Expected behavior:
Fast pass should be pass for disable combo-box, like other office UI fabric disable component

**Comment by aneeshack4** [comment]
Thanks for reaching out to us... @MdAmeerMind. It looks to be correct to me when I actually go into high contrast mode: 
![image](https://user-images.githubusercontent.com/4161791/82348161-1d24ac00-99be-11ea-9d9d-b675d2e367c3.png)
You're most likely getting this error because you're running the Accessibility Insights tool on a Chrome which doesn't support high contrast mode - testing contrast while high contrast mode is not applied... If you see anything actually missing with the high contrast styling of this control in disabled state, then we can look at this further.

**Comment by ecraig12345** [comment]
I think they just meant this is an issue with color contrast in general, not with high contrast mode specifically.

In any case, disabled components are actually not subject to color contrast requirements, and Accessibility Insights results should reflect that (the fact that it doesn't is a bug on their side). This was previously reported under #9909, which was closed since it's due to a bug with the tool--being tracked under https://github.com/microsoft/accessibility-insights-web/issues/981 and https://github.com/dequelabs/axe-core/issues/2090.

I'm going to close this since it's a duplicate.
